examples/supply_chain_security/python_env.yaml

```diff
@@ -2,5 +2,7 @@ python: "3.9"
 build_dependencies:
   - pip
 dependencies:
-  - --require-hashes
-  - -r requirements.txt
```

Comment:
The old `python_env.yaml` config pins all lib versions inside `requirements.txt`, this causes the suite runs against an old and pinned MLflow version. So I update it here.